tests/test_client.py

```diff
@@ -720,6 +770,52 @@ def update_categories(self, obj_id, version):
                 'idempotency_key': str(uuid.uuid4())}
         return self.post_category(body)
 
+    def update_employees_v1(self, obj):
+        if self.env_is_sandbox():
+            raise RuntimeError("The Square Environment is set to {} but must be production.".format(self._environment))
+
+        base_v1 = "https://connect.squareup.com/v1/me/"
+        endpoint = "employees"
+        employee_id = obj.get('id')
+        full_url = base_v1 + endpoint + "/" + employee_id
+
+        uid = self.make_id('employee')[1:]
+        data = {'first_name': uid,
+                'last_name': obj.get('last_name')}
+
+        resp = requests.put(url=full_url, headers=self.get_headers(), json=data)
+        if resp.status_code != 200:
+            raise Exception(resp.text)
+        return resp.json()
+
+    def update_roles_v1(self, obj):
```

Comment:
This and the above method are nearly identical. Can you create a method they both call? I think it'll save us time in the future if we ever have to make changes to the way we call v1 endpoints.

Reply:
I can, should I wait to until settlements is done though? Rule of 3

Reply:
Nevermind I'll just do it